test/hotspot/jtreg/vmTestbase/nsk/jvmti/GetCurrentContendedMonitor/contmon001.java

```diff
@@ -55,8 +54,17 @@ public static void main(String[] args) {
         System.exit(run(args, System.out) + 95/*STATUS_TEMP*/);
     }
 
+    public static void doSleep() {
+        try {
+            Thread.sleep(10);
+        } catch (Exception e) {
+            throw new Error("Unexpected " + e);
+        }
+    }
+
     public static int run(String argv[], PrintStream ref) {
         out = ref;
+        doSleep(); // If we need to load any classes to execute doSleep(), do it now.
```

Comment:
Well intentioned but not really useful. The classes used on the normal execution path are already loaded during VM initialization. The exceptional paths can still lead to class loading/linking/synchronization, so this preemptive call doesn't help that case anyway.

Reply:
I can remove the 'dummy' calls in each test?